content/guides/output-management.md

```diff
@@ -221,6 +222,20 @@ __webpack.config.js__
   };
 ```
 
+__src/index.html__
```

Comment:
I'm just wondering if this is actually used, because of the `HtmlWebpackPlugin`.

Let me run a build with it and see if it's used or not.